src/ggrc/models/issuetracker_issue.py

```diff
@@ -193,8 +194,7 @@ def update_from_dict(self, info):
     self.issue_id = info['issue_id']
     self.issue_url = info['issue_url']
 
-    if info.get('due_date'):
-      self.due_date = info.get('due_date')
+    self.due_date = info.get('due_date')
```

Comment:
Please take a look at the suggest below

```suggestion
    if 'due_date' in info:
      self.due_date = info['due_date']
```